src/Tests/EmbedButtonAdminTest.php

```diff
@@ -99,4 +100,40 @@ public function testEmbedButtonAdmin() {
     $this->assertNoText($name, 'Test embed_button does not appears on the list page');
   }
 
+  /**
+   * Tests the embed_button and file usage integration.
+   */
+  public function testEmbedButtonIcon() {
```

Comment:
I think this test will be more suited if it were moved to EmbedButtonCrudTest since we're not really testing administrative interface here.